sphinx_gallery/scrapers.py

```diff
@@ -41,7 +41,9 @@ def _import_matplotlib():
 
     filterwarnings("ignore", category=UserWarning,
                    message='Matplotlib is currently using agg, which is a'
-                           ' non-GUI backend, so cannot show the figure.')
+                           ' non-GUI backend, so cannot show the figure.'
+                           '|(\n|.)*is non-interactive, and thus cannot be' 
```

Comment:
```suggestion
                           '|(\n|.)*is non-interactive, and thus cannot be'
```

Reply:
Trailing whitespace!

<img width="127" alt="Screenshot 2023-08-10 at 6 44 09 PM" src="https://github.com/sphinx-gallery/sphinx-gallery/assets/2365790/75cf9074-8c45-486f-9432-b790dcd4b4b8">


And yes SG should definitely get on board with `black` and `pre-commit.ci` etc. someday...